src/Microsoft.ML.ImageAnalytics/VectorToImageTransform.cs

```diff
@@ -195,7 +195,7 @@ public ColInfoEx(ModelLoadContext ctx)
                 Interleave = ctx.Reader.ReadBoolByte();
             }
 
-            public void Save(ModelSaveContext ctx)
+            internal void Save(ModelSaveContext ctx)
```

Comment:
>internal [](start = 12, length = 8)

Same notes about `ColInfoEx` being a private class, no need to internalize its methods.